gradle/libs.versions.toml

```diff
@@ -17,13 +17,13 @@ desugaring = "2.0.4"
 
 # Android Gradle Plugin
 # https://developer.android.com/studio/releases/gradle-plugin.
-agp = "8.2.0"
+agp = "8.2.1"
 
 # AndroidX
 # https://androidx.tech.
 activity = "1.8.2"
-core = "1.13.0-alpha02"
-lifecycle = "2.7.0-rc02"
+core = "1.13.0-alpha03"
```

Comment:
what features do you need from using the alpha version?

Reply:
Yes